### ISSUE TITLE
Shouldn't there be separately declared losses?

### ISSUE THREAD

**Issue by seanswyi** [ShannonAI/mrc-for-flat-nested-ner#85]
Hi, thanks for your work. I'm just curious regarding the loss calculation inside the trainer module. It seems like you're declaring the loss function once (e.g., `self.bce_loss = nn.BCEWithLogitsLoss()`) but shouldn't you be declaring a loss object for the start, end, and matching logits? If I'm misunderstanding something please feel free to let me know. Thanks!
